test/upgrades/v2.0.1/upgrade.v2.0.1.t.sol

```diff
@@ -126,10 +113,10 @@ contract UpgradeBehaviorTest is Test {
         assert(account.getConditionalOrder(0).marketKey == sETHPERP);
         assert(account.getConditionalOrder(0).targetPrice == 1000 ether);
         assert(account.freeMargin() == 0);
-        assert(account.delegates(KWENTA_TREASURY));
+        assert(account.delegates(DELEGATE));
 
         // check execute() can be locked
-        vm.prank(KWENTA_TREASURY);
+        vm.prank(OPTIMISM_GOERLI_KWENTA_ADMIN_DAO_MULTI_SIG);
         settings.setAccountExecutionEnabled(false);
 
         // try to execute a command
```

Comment:
**Thought:** This seems like it will 100% work, but I also wonder if it is worth executing a command successfully in the test once the upgrade is completed.

Reply:
+1 this. Prob better to validate a happy case here